plugins/com.google.cloud.tools.eclipse.util/src/com/google/cloud/tools/eclipse/util/DependencyResolver.java

```diff
@@ -108,4 +112,44 @@
     repositories.add(repository);
     return repositories;
   }
+
+  public static Collection<Dependency> getManagedDependencies(String groupId, String artifactId,
+      String version, IProgressMonitor monitor) throws CoreException {
+    
+    // todo we'd prefer not to depend on m2e here
+    final Artifact artifact =
```

Comment:
Seems like `final` is unnecessary starting with Java 8.

Reply:
done